.github/workflows/ci.yml

```diff
@@ -11,22 +10,23 @@ on:
       - main
 
 jobs:
-  run-linters:
-    name: Run linters
+  test:
+    name: 🧪 Test
     runs-on: ubuntu-latest
 
     steps:
       - name: Check out Git repository
         uses: actions/checkout@v2
       - name: Set up Node.js
-        uses: actions/setup-node@v1
+        uses: actions/setup-node@v3
         with:
-          node-version: 14
-
+          node-version: 16
```

Comment:
If you upgrade node to 16 here, would you also update it in `main.yml` so all of our builds use the same node version?